### DISCUSSION TITLE
rabbitmqctl list_amqp10_connections does not list AMQP v1.0 connections

### DISCUSSION THREAD

**Discussion by motmot80** [rabbitmq/rabbitmq-server#5511] [Other]
# Environment:
- RabbitMQ: 3.9.13
- Erlang: 24.2.1
- OS: Windows, Linux RHEL, Linux Ubuntu

# Steps to reproduce:
1. Connect AMQP v1.0 (TLS or non-TLS) clients to RabbitMQ
![grafik](https://user-images.githubusercontent.com/1308503/184554168-6eec0979-68ee-4f53-96ad-f489e8d026fc.png)
2. Execute `rabbitmqctl list_amqp10_connections `

# Expected:
Connections are shown.

# Actual:
List is empty.

`rabbitmqctl list_amqp10_connections ` is showing:
`Listing AMQP 1.0 connections ...`

`rabbitmqctl list_channels` is showing:
```
Listing channels ...
pid     user    consumer_count  messages_unacknowledged
<rabbit@TESTNODE.1660241656.15715.2>     guest   1       0
```

**Comment by michaelklishin** [comment]
This version is almost ten patch releases behind. Please try 3.9.22 and 3.10.7 first.

**Reply by motmot80** [reply]
We'll try to update to v3.11.1 ; )

https://github.com/rabbitmq/rabbitmq-server/issues/4238
